eZ/Publish/Core/FieldType/FieldType.php

```diff
@@ -271,7 +271,7 @@ public function applyDefaultSettings(&$fieldSettings)
      */
     protected function getSortInfo(Value $value)
     {
-        throw new \RuntimeException('Not implemented, yet.');
+        return false;
```

Comment:
Imo null would be better

Reply:
> Imo null would be better

@pawbuj, I agree but this has deeper consequences. Right now `false` is internally treated as "no sort info".

@adamwojs Does this PR block anything?
Asking because I think we need to rectify this and indeed use `null`. 
Use case: you have boolean FT which you want to sort by false first.

Reply:
> @adamwojs Does this PR block anything?

Generic Field Type depends on this change. 

> Asking because I think we need to rectify this and indeed use null.
>Use case: you have boolean FT which you want to sort by false first.

`null` is also allowed value here, see `\eZ\Publish\Core\FieldType\Null\Type::getSortInfo`. Using `false` is just a non formal convention for non-sortable FT.

Reply:
> `null` is also allowed value here, see `\eZ\Publish\Core\FieldType\Null\Type::getSortInfo`. Using `false` is just a "non-formal" convention for non-sortable FT.

What are the implications of returning `null` then?

Reply:
There are no implications at this point. Value returned by `getSortInfo` is passed as `\eZ\Publish\SPI\Persistence\Content\FieldValue::$sortKey` to `\eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter` implementation and  needs to be handled there. 

Suggestion applied in https://github.com/ezsystems/ezpublish-kernel/pull/2677/commits/fa8fc7598e3069e2ff9922e6c5085d67ef6af4fb